src/_includes/layouts/base.njk

```diff
@@ -4,7 +4,9 @@
     <meta charset="utf-8">
     <title>{% if '/' === page.url %}V8 JavaScript engine{% else %}{{ (renderData.title or title) | markdown | striptags | decodeHtmlEntities }} · V8{% endif %}</title>
     <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="color-scheme" content="dark light">
     <link rel="stylesheet" href="/_css/main.css">
+    <noscript><style>:root{display:block}</style></noscript>
```

Comment:
`<html>` gets `class=js` when JS is enabled (see `<script>document.documentElement.className+=' js'</script>` a couple of lines down). Can we leverage that within `main.css` instead of adding `<noscript><style>` here?

Reply:
Addressed in https://github.com/v8/v8.dev/pull/254/commits/3b32524d665aa734e362db26b9040280bd3881d0.